lib/Source.ml

```diff
@@ -116,6 +116,12 @@ let is_long_pexp_open source {pexp_desc; _} =
       contains_token_between source ~from ~upto Parser.IN
   | _ -> false
 
+let is_long_ppat_open source {ppat_desc; _} =
+  match ppat_desc with
+  | Ppat_open ({loc= from; _}, {ppat_loc= upto; _}) ->
+      contains_token_between source ~from ~upto Parser.IN
```

Comment:
I don't expect this token in a pattern. What's a "long" ppat open ?

Reply:
Ah yes the `let open` syntax is not possible indeed, I didn't even check, I will remove this